sqldb/rust/Cargo.toml

```diff
@@ -0,0 +1,34 @@
+[package]
+name = "wasmcloud-interface-sqldb"
+version = "0.1.0"
+description = "interface for actors to issue http/https requests (wasmcloud:httpclient)"
+repository = "https://github.com/wasmcloud/interfaces"
+homepage = "https://github.com/wasmcloud/wasmcloud"
+keywords = ["wasmcloud","wasm","actor","webassembly","capability"]
+categories = [ "api-bindings","network-programming","wasm" ]
+authors = [ "wasmcloud Team" ]
+edition = "2018"
+license = "Apache-2.0"
```

Comment:
should also specify the `readme` field to indicate the location of the `README` to display on crates dot io.